sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TableCreationTests.cs

```diff
@@ -110,6 +112,7 @@ public static void Call([Table(TableNameExpression)] out T entity)
         [GenericTestCase(typeof(JObject))]
         [GenericTestCase(typeof(StructTableEntity))]
         [GenericTestCase(typeof(PocoTableEntity))]
+        [GenericTestCase(typeof(ExplicitlyImplementedITableEntity))]
```

Comment:
I don't think that is a valid test since this operates on an explicit type rather than a type inferred from an instance. There is a test that does feed an explicit `ITableEntity` to the binding elsewhere.

Reply:
The test I'm thinking of binds to an ITableEntity, but the underlying instance is not TableEntity.